content/general/concepts/white-spaces/whitespace.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'Whitespace'
+Description: 'Whitespace is spaces and blank lines in programing.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Formatting'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+**Whitespace** is spaces or blank lines in programming. Whitespace is used to format code to improve its readability. 
+
+## Example
+
+The below code is hard to read since it has no gaps and it also does not have indention with spaces so people can not easily see what is nested within what.
```

Comment:
```suggestion
The code example below is hard to read since there is little to no whitespace. It also does not have indention with spaces so people can not easily see what is nested within what.
```